ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Set cluster variables
+# CLUSTER_NAME=$(cat "${SHARED_DIR}/CLUSTER_NAME")
+# CLUSTER_DOMAIN="${CLUSTER_DOMAIN:-release-ci.cnv-qe.rhood.us}"
+BIN_FOLDER=$(mktemp -d /tmp/bin.XXXX)
+curl -L https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/stable/openshift-client-linux.tar.gz \
+    | tar -C "${BIN_FOLDER}" -xzf - oc
+
+# Exports
+# export CLUSTER_NAME CLUSTER_DOMAIN
+export PATH="${BIN_FOLDER}:${PATH}"
+
+# Unset the following environment variables to avoid issues with oc command
+unset KUBERNETES_SERVICE_PORT_HTTPS
+unset KUBERNETES_SERVICE_PORT
+unset KUBERNETES_PORT_443_TCP
+unset KUBERNETES_PORT_443_TCP_PROTO
+unset KUBERNETES_PORT_443_TCP_ADDR
+unset KUBERNETES_SERVICE_HOST
+unset KUBERNETES_PORT
+unset KUBERNETES_PORT_443_TCP_PORT
+
+set -x
+START_TIME=$(date "+%s")
+
+poetry run pytest --junitxml "${ARTIFACT_DIR}/xunit_results.xml" --pytest-log-file="${ARTIFACT_DIR}/tests.log" -o cache_dir=/tmp --tc=hco_subscription:kubevirt-hyperconverged
```

Comment:
Are we creating subscription named `kubevirt-hyperconverged`?

Reply:
Yes, 

https://github.com/openshift/release/blob/master/ci-operator/step-registry/interop-tests/deploy-cnv/interop-tests-deploy-cnv-commands.sh#L39

Reply:
@rabin-io that does not seem right ```kind: Subscription
metadata:
  name: hco-operatorhub
  namespace: $CNV_INSTALL_NAMESPACE```. The name seems to be `hco-operatorhub` based on the link you shared. In that case the command you used to override hco_subscription `--tc=hco_subscription:kubevirt-hyperconverged` is not necessary, as we already have it correct in https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/global_config.py#L6

Reply:
I will check this once I refactor the jobs to use our cluster profile again.